src/db/tripledb.plt

```diff
@@ -96,5 +96,10 @@ test('tripledb tell triple with time scope extension'):-
     tripledb_tell(test_datatype:'Lecturer4', test_datatype:'last_name', 'Spiendler', T_S1),
     assert_true(tripledb_ask(test_datatype:'Lecturer4', test_datatype:'last_name', 'Spiendler', T_S1, _, _)).
 
+% test for unit
+test('tripledb tell triple with Unit'):-
+    tripledb_tell(test_datatype:'Lecturer4', test_datatype:'height', unit(double(2.1),'meter')),
```

Comment:
you can also wrap this call in *assert_true*. Then you get more informative message in case the test fails.

Reply:
done.